pkg/R/schema.R

```diff
@@ -0,0 +1,169 @@
+#' structType
```

Comment:
A one line comment on top with a description of what is in this file would be good